src/main/java/com/revature/services/RSSService.java

```diff
@@ -137,7 +137,28 @@ public User login(RSSUserDTO u) {
 	 * @Author Kei
 	 */
 	public int getPoints(int id) {
-		return id;
+		String uri =  "http://ec2-34-203-75-254.compute-1.amazonaws.com:10001/account/account";
```

Comment:
We will need this endpoint to be injected at some point in the future.  For now, supply a value somewhere in the application.yml with an arbitrary name, then reference the key here using @Value("${key.name}") to inject the endpoint.